FR: Add missing support for Apple Silicon Mac Catalyst apps when integrating Firebase manually with xcframeworks.

<!-- DO NOT DELETE
validate_template=true
template_path=.github/ISSUE_TEMPLATE/feature_request.md
-->

### Feature proposal

* Add the missing Apple Silicon slices to the maccatalyst variant of the xcframeworks in Firebase.zip.

The Release 7.1.0 of Firebase.zip containing all the .xcframeworks for manual integration does not contain an arm64 slice for the maccatalyst variant.
As a result, when creating an archive for "Any Mac (Apple Silicon, Intel)", linking fails with the following errors:
```
Undefined symbols for architecture arm64:
  "_OBJC_CLASS_$_FIRApp", referenced from:
      objc-class-ref in XXX
  "_OBJC_CLASS_$_FIRCrashlytics", referenced from:
      objc-class-ref in XXXX
  "_OBJC_CLASS_$_FIRDatabase", referenced from:
      objc-class-ref in XXX
  "_OBJC_CLASS_$_FIRRemoteConfig", referenced from:
      objc-class-ref in XXX
  "_OBJC_CLASS_$_FIRTransactionResult", referenced from:
      objc-class-ref in XXX
...
```

--- a/FirebaseCore/CHANGELOG.md
+++ b/FirebaseCore/CHANGELOG.md
@@ -3,7 +3,7 @@
 - [added] Introduced FirebaseMLModelDownloader.
 - [fixed] Fixed missing doc comment in `FirebaseVersion()`. (#7506)
 - [changed] Minimum required Xcode version for Zip and Carthage distributions changed to 12.2 (was 12.0).
-- [added] The zip distribution now includes Catalyst simulator slices. (#7007)
+- [added] The zip distribution now includes Catalyst arm64 simulator slices. (#7007)
 
 # FirebaseCore 7.6.0
 - [fixed] Fixed build warnings introduced with Xcode 12.5. (#7431)

--- a/FirebaseCore/CHANGELOG.md
+++ b/FirebaseCore/CHANGELOG.md
@@ -1,7 +1,9 @@
 # FirebaseCore 7.7.0
 - [changed] Deprecated FirebaseMLModelInterpreter and FirebaseMLVision.
 - [added] Introduced FirebaseMLModelDownloader.
-- [fixed] Fixed missing doc comment in `FirebaseVersion()` (#7506).
+- [fixed] Fixed missing doc comment in `FirebaseVersion()`. (#7506)
+- [changed] Minimum required Xcode version for Zip and Carthage distributions changed to 12.2 (was 12.0).
+- [added] The zip distribution now includes Catalyst simulator slices. (#7007)
 
 # FirebaseCore 7.6.0
 - [fixed] Fixed build warnings introduced with Xcode 12.5. (#7431)

